src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -138,12 +164,71 @@ public void setServerUrl(String serverUrl) {
     this.serverUrl = serverUrl;
   }
 
+  public String getAnetDictionaryName() {
+    return anetDictionaryName;
+  }
+
+  public void setAnetDictionaryName(String anetDictionaryName) throws Exception {
+    this.anetDictionaryName = anetDictionaryName;
+    this.loadDictionary();
+  }
+
   public Map<String, Object> getDictionary() {
     return dictionary;
   }
 
   public void setDictionary(Map<String, Object> dictionary) {
-    this.dictionary = Collections.unmodifiableMap(dictionary);
+    this.dictionary = dictionary;
```

Comment:
Why this change from unmodifiable to modifiable?

Reply:
I made changes to the code a couple of times, I think it's up to that time.
I probably thought of another solution at the beginning and so I changed it but I couldn't remember the exact reason.
With the current code structure I have modified it as it is before